WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
Another option:

    last_unskipped_index = solve_times.rindex{|st|  not st.skipped? } || 0
    first_skipped_index = solve_times..index{|st| st.skipped? } || 999
    errors.add(:base, "Skipped solves must all come at the end.") if last_unskipped_index > first_skipped_index